### ISSUE TITLE
Publish package to npm

### ISSUE THREAD

**Issue by ProLoser** [nwcell/ics.js#45]
I use npm for developing client side. Do you think you could publish this package on there?

Also, you should probably move your `package.json` dependencies to `devDependencies`

**Comment by lucis** [comment]
One temporary solution is using cdnJS: 
https://cdn.jsdelivr.net/gh/nwcell/ics.js@0.2.0/ics.deps.min.js

**Comment by Jones-S** [comment]
But an npm package would be much nicer!

**Comment by PerpetualWar** [comment]
yep, it will mostly be used with some bundler as webpack. I found on npm ics-browser-gen, which seems to be this library, but importing it as ES module fails with message it is not a function.

Any idea how to import this lib inside front end framework if ES Module imports are not working?

**Comment by ickas** [comment]
@PerpetualWar, any news related to this?
